android/android_browser_tests.gni

```diff
@@ -108,6 +108,7 @@ android_only_test_sources = [
 android_only_test_deps = [
   ":test_support",
   "//android_webview/common",
+  "//brave/browser/android/background_video:test_support",
```

Comment:
I'm actually not sure why `android_only_test_deps` even exists, it doesn't really make sense and these deps should just do directly in the `is_android` block in `brave_browser_tests` and this can be added directly there as well. `android_browser_tests.gni` is really about tests that should run on android, but currently don't because they have to be updated.

Reply:
I'm going to put `android_only_test_deps` on my list of things to fix and add some better docs